package.json

```diff
@@ -502,9 +502,18 @@
                 "default": "The process name to attach to. If this is used, 'processId' should not be used."
               },
               "processId": {
-                "type": "integer",
-                "description": "The process id to attach to. If this is used, 'processName' should not be used.",
-                "default": 0
+                "anyOf": [ 
+                    {
+                      "type": "string",
+                      "description": "Lists running processes to attach to. If this is used, 'processName' should not be used.",
```

Comment:
How about: "The process id to attach to. Use "${command.pickProcess}" to get a list of running processes to attach to. If this is used, 'processName' should not be used.